fe/fe-core/src/main/java/org/apache/doris/alter/AlterLSCHelper.java

```diff
@@ -79,7 +80,16 @@ public void enableLightSchemaChange() throws DdlException {
         final Map<Long, PFetchColIdsRequest> params = initParams();
         final PFetchColIdsResponse response = callForColumnIds(params);
         updateTableMeta(response);
-        modifyTableProperty();
+        writeEditLog(response);
```

Comment:
It is unnecessary add a function `writeEditLog()` here

Reply:
I think we could keep `writeEditLog()` here so that we can omit the writeEditLog related logic in replay process, and just use `updateTableMeta()` then.